src/Products/urban/notice/parcel.py

```
@@ -71,3 +71,26 @@ def puissance(self):
+
+    @property
+    def division_text(self):
+        return self._get_data("division")
+
+    @property
+    def resolved_division(self):
+        code = self._get_data("codeDivision")
+        if code:
+            return code
+        #do mapping 
+        division_name = division_name.strip().upper()
```

Ruff (0.15.9)

[error] Undefined name `division_name`

(F821)